lib/lhm.rb

```diff
@@ -82,15 +83,26 @@ def cleanup_current_run(run, table_name, options = {})
     Lhm::Cleanup::Current.new(run, table_name, connection, options).execute
   end
 
-  def setup(connection)
-    @@connection = Lhm::Connection.new(connection: connection)
+  # Setups DB connection
+  #
+  # @param [ActiveRecord::Base] connection ActiveRecord Connection
+  # @param [Hash] connection_options Optional options (defaults to: empty hash)
+  # @option options [Boolean] :reconnect_with_consistent_host
```

Comment:
Would there be a scenario where someone wouldn't want to reconnect with with consistent host ? The reason I am asking here is because we could get rid of the optional variable `reconnect_with_consistent_host` if it is always being set to true.

Reply:
Right now we'll use it as a feature flag, so it is `false` by default (so that people would not inadvertently try to reconnect with ProxySQL without having added the required rules)

Reply:
fixed